Diff/diff.py

```diff
@@ -1,14 +1,19 @@
-import codecs
+from __future__ import annotations
+
 import difflib
-import os.path
+import os
 import time
 
 import sublime
 import sublime_plugin
 
+import typing
+
 
-def splitlines_keep_ends(text):
-    lines = text.split('\n')
+def splitlines_keep_ends(
+    text: str
+) -> typing.List[str]:
```

Comment:
Why adding line breaks?

IMO there should be either explicit line limit defined that would dictate whether line should be broken or, if there is no limit, then just don't break lines arbitrarily.

There is benefit to having as much code as possible to fit within the editor view.